inference-engine/tests/functional/inference_engine/transformations/pad_fusion.cpp

```diff
@@ -122,7 +122,7 @@ TEST(TransformationTests, PadFusionAvgPoolDontExcludePad) {
     ASSERT_TRUE(res.first) << res.second;
 }
 
-TEST(TransformationTests, PadFusionMaxPool) {
+TEST(TransformationTests, DISABLED_PadFusionMaxPool) {
```

Comment:
Why did you disable this test?

Reply:
Because I disabled this fusing. There was a problem with incorrect padding values resulting in an accuracy drop
Is this test should be deleted?